terminology/README.md

```diff
@@ -0,0 +1,21 @@
+# Terminology
+
+The language we use has power. Being as concise and descriptive as possible helps to lessen ambiguity. This leads to shared understanding and better outcomes.
+
+## Terms
+
+* Use `blocklist`, and not `blacklist`.
+* Use `allowlist`, and not `whitelist`.
+* Avoid using `master` and `slave`. 
+	* For GitHub, use a term such as `main`, or `publish`, or `trunk`.
+	* For databases, use `write database` and `read replica`.
+	* For other scenarios, use terms such as `parent` and `child` or `leader` and `follower`.
+* Use terms such as `excused` or `exempted` in place of `grandfathered`.
+* Use `negative space`, and not `white space`.
+* Use `billable hours`, and not `man hours`.
+
+[google-io-video]: https://www.youtube.com/watch?v=mLjxXPHuIJo
+[web-dev-fast]: https://web.dev/fast
+[lighthouse-budgets]: https://bitsofco.de/your-first-performance-budget-with-lighthouse/
+[css-wizardry]: https://csswizardry.com/archive/
+[calibre-impact]: https://calibreapp.com/blog/visualise-performance-impact-between-deploys
```

Comment:
Should these links be removed?

Reply:
This is what happens when you think you hit push, but don't :sweat\_smile: